README.md

```diff
@@ -59,7 +59,7 @@ Useful Helm Client Commands:
 
 ## License
 
-Copyright &copy; 2023 VMware Inc
+Copyright &copy; 2023 VMware, Inc.
 
 Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
```

Comment:
This file is not using the [Apache 2.0 license header](https://spdx.org/licenses/Apache-2.0.html) as is, please check the break lines.